tensorflow/compiler/mlir/tools/kernel_gen/transforms/tf_to_jit_invocations.cc

```diff
@@ -153,7 +170,7 @@ struct TFToI64JITInvocationForLargeTensorsPattern : public RewritePattern {
       // Create JIT execute op.
       auto jit_execute_op = b.create<tf_framework::JITExecuteOp>(
           loc, op->getResultTypes().front(), /*ctx=*/Value(),
-          jit_compile_op.getResult(), arg);
+          jit_compile_op.getResult(), op->getOperands());
```

Comment:
Maybe add an assert here to document that expect one or two operands here

Reply:
added.